examples/testmonitor/README.md

```diff
@@ -0,0 +1,31 @@
+SystemLink Test Monitor Service API Examples
+====================================
+
+This directory holds self-contained example projects for the SystemLink Test Monitor service API. To make finding relevant examples easier, a summary of each one is listed below.
+
+Examples for other SystemLink Client APIs are available in the
+[root examples directory](..).
+
+Running an Example
+------------------
+
+Unless otherwise specified by the example's README, each example is run in the
+same way:
```

Comment:
Complete the following steps to run each example, unless the README specifies different instructions:

Reply:
Updated in a later commit:

"To run the example, use the following command:"